<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VecMapDialog</class>
 <widget class="QWidget" name="VecMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Layer</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="6">
    <widget class="QFrame" name="Panel1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="12" column="0">
       <widget class="QCheckBox" name="cBVisible12">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="btnColor12F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="cBVisible7">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnColor3">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="btnColor9F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="cBVisible5">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="btnColor9">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cBVisible1">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="rBLayer9">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QRadioButton" name="rBLayer10">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cBVisible3">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnColor1">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnColor4F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="cBVisible10">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rBLayer2">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QRadioButton" name="rBLayer11">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnColor3F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnColor5">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QPushButton" name="btnColor12">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="cBVisible8">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Label3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnColor5F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="Label2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="btnColor8">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="btnColor10">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="btnColor8F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rBLayer1">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cBVisible2">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="btnColor11">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QRadioButton" name="rBLayer8">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnColor6">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnColor4">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="btnColor7">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="btnColor7F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="cBVisible6">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rBLayer3">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QRadioButton" name="rBLayer6">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="cBVisible9">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnColor2">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QRadioButton" name="rBLayer12">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="cBVisible11">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Label1">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cBVisible4">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnColor1F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="btnColor10F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QPushButton" name="btnColor11F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnColor2F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnColor6F">
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QRadioButton" name="rBLayer7">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fill</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="rBLayer5">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="rBLayer4">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnUp">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Up</string>
     </property>
     <property name="icon">
      <iconset resource="rtkplot_qt.qrc">
       <normaloff>:/buttons/up2</normaloff>:/buttons/up2</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="btnDown">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Down</string>
     </property>
     <property name="icon">
      <iconset resource="rtkplot_qt.qrc">
       <normaloff>:/buttons/down</normaloff>:/buttons/down</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btnApply">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Apply</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="btnClose">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rtkplot_qt.qrc"/>
 </resources>
 <connections/>
</ui>
